doc/articles/contributing/guidelines/creating-tests.md

```diff
@@ -36,16 +36,16 @@ These are the main types of tests in Uno:
 
 UI tests can mimic the actions of a user:
 
- - tapping buttons and other UI elements
- - entering keyboard input
- - scrolling, swiping, and other gestures
+* tapping buttons and other UI elements
+* entering keyboard input
+* scrolling, swiping, and other gestures
```

Comment:
This is very confusing. Are we unifying on `*` or `-`? There are files where you switch `*` to `-` and there are the opposite. I think it's the autofix picking up whatever appears first in the file? But if we want consistency, we should have it for all files. Otherwise, we can just disable this rule to avoid unnecessarily large changes

Reply:
@Youssef1313 Markdownlint only cares that things are consistent within a given document. If you have a preference, I can set that within the markdownlint preferences and update all bullets.

What is the project's preference?